guides/micronaut-security-session/java/src/main/java/example/micronaut/LoginAuthController.java

```diff
@@ -1,29 +1,24 @@
 package example.micronaut;
 
-import io.micronaut.http.MediaType;
 import io.micronaut.http.annotation.Controller;
 import io.micronaut.http.annotation.Get;
-import io.micronaut.http.annotation.Produces;
 import io.micronaut.security.annotation.Secured;
 import io.micronaut.security.rules.SecurityRule;
 import io.micronaut.views.View;
 
 import java.util.Collections;
-import java.util.HashMap;
 import java.util.Map;
 
 @Secured(SecurityRule.IS_ANONYMOUS) // <1>
 @Controller("/login")  // <2>
 public class LoginAuthController {
 
-    @Produces(MediaType.TEXT_HTML)
     @Get("/auth") // <3>
     @View("auth") // <4>
     public Map<String, Object> auth() {
-        return new HashMap<>();
+        return Collections.emptyMap();
```

Comment:
We should not return an inmutable Map as the model. Because, we support a feature called [ViewModelProcessors](https://micronaut-projects.github.io/micronaut-views/latest/guide/#custom) which allows you to decorate a model. That it is to say add stuff to the map.